### ISSUE TITLE
Adaptation of BlackVIP to custom datasets

### ISSUE THREAD

**Issue by HaokunChen245** [changdaeoh/BlackVIP#4]
Dear authors,

thanks for the great work and the very interesting method! 

What hyperparameter is crucial (or how did you tune them) when adapting BlackVIP to custom datasets? Because when I use the default hyperparameters the training seems to diverge and the loss couldn't go down.

Thanks in advance.

**Comment by changdaeoh** [comment]
Thanks for your interest, and sorry for my late response!

after setting gamma=0.1 and spsa_c=0.005
1. find the proper `spsa_a` and `alpha` (analogy to learning rate and its decaying factor) 
2. tune `p_eps` across 0.3 ~ 1.0
3. if still doesn't work, alter `moms` from 0.9 to 0.5, 0.1, and so on. (rarely, weak momentum brings better results)

The learning curves of black-box methods are highly jagged.
But if you choose proper parameters, the training loss will decrease anyway!
